src/notebook/notebookFeature.ts

```diff
@@ -122,15 +115,14 @@ export class JuliaNotebookFeature {
             await currentKernel.queueCell(cell)
         }
     }
-    private getKernelSpecNameAndVersion(notebook: vscode.NotebookDocument): {name:string; version: string} {
+    private getKernelSpecName(notebook: vscode.NotebookDocument): string {
         const metadata = (notebook.metadata as JupyterNotebookMetadata)?.custom.metadata;
         const kernelspecName = metadata?.kernelspec?.name || '';
-        const version = metadata?.language_info?.version || '';
-        return this.isJuliaNotebook(notebook) ? { name: kernelspecName, version } : {name:'', version:''}
+        return this.isJuliaNotebook(notebook) ? kernelspecName : ''
     }
     private updateNotebookWithSelectedKernel(notebook: vscode.NotebookDocument, name: string, version: string) {
         // Dont edit in place, create a copy of the metadata.
-        const nbmetadata: JupyterNotebookMetadata = JSON.parse(JSON.stringify((notebook.metadata || { custom: { metadata: {}}})));
+        const nbmetadata: JupyterNotebookMetadata = JSON.parse(JSON.stringify((notebook.metadata || { custom: { metadata: {} } })));
```

Comment:
What formatter do you use, i use is prettier and it seems to format the whole document & don't know whether my code is inline with the formatting/style that you follow in this repo.